process/cg-transition.html

```diff
@@ -1,102 +1,237 @@
-
-<address><a href="https://www.w3.org/People/Dom/">Dominique Hazael-Massieux</a> for
-<a href="https://www.w3.org/People/LeHegaret/">Philippe Le Hégaret</a>, guidebook editor<br>
-             <a href="mailto:plh@w3.org">plh@w3.org</a><br>
+    }
+    </style>
+  </head>
+  <body>
+    <div id="header">
+      <span class="logo"><a href="/"><img src="/Icons/WWW/w3c_home_nb" alt=
+      "W3C" height="48" width="72"></a></span>
+      <div class="breadcrumb">
+        <a href="/Member/">Member</a> → <a href="/Guide/">The Art of
+        Consensus</a> →
+        <h1>
+          How to transition work from a Community Group to a Working Group
+        </h1>
+      </div>
+      <p class="baseline">
+        This <strong>Guidebook</strong> is the collected wisdom of the W3C
+        Group Chairs and other collaborators.
+      </p>
+    </div>
+    <nav class="toc">
+      <h4>
+        Also On This Page →
+      </h4>
+    </nav>
+    <p>
+      From time to time a Community Group that has been incubating a technology
+      seeks to advance it to the W3C Recommendation track in a Working Group.
+      Our goal through this document is to ease that transition.
+    </p>
+    <h2>
+      Preparation
+    </h2>
+    <p>
+      The following questions can help a Community Group to prepare for the
+      transition of work to a Working Group.
+    </p>
+    <h3>
+      Is the work ready for standardization?
+    </h3>
+    <p>
+      Please see elsewhere in the Guide: <a href=
+      "https://www.w3.org/Guide/standards-track/">W3C Recommendation Track
+      Readiness Best Practices</a>.
+    </p>
+    <h3>
+      What is the target Working Group and the relation to the CG?
+    </h3>
+    <p>
+      If the Working Group does not yet exist, the Community Group can prepare
+      by <a href="https://www.w3.org/Guide/process/charter.html">drafting a
+      Working Group charter</a>. Otherwise, if the group exists but the new
+      work is not within its scope W3C needs to <a href=
+      "https://www.w3.org/Guide/process/charter.html">recharter the group</a>
+      to accommodate the new work.
+    </p>
+    <p>
+      The charter of the target Working Group should address two topics:
+    </p>
+    <ul>
+      <li>The list of any deliverables transferred from a Community Group.
+      </li>
+      <li>The nature of the working relationship between the Working Group and
+      the Community Group (if any) for any deliverable originated by the
+      Community Group.
+      </li>
+    </ul>
+    <p>
+      @@@TODO extract more from <a href=
+      "https://github.com/w3c/charter-drafts/issues/262">charter template
+      discussion</a>
+    </p>
+    <p>
+      Community Group chairs should keep the Community Group participants aware
+      of the progress on the transition roadmap, in particular if it involves
+      chartering or re-chartering a Working Group.
+    </p>
+    <h3>
+      Is there a record of Community Group consensus for the transition?
```

Comment:
I had purposefully not used consensus as a criteria, since I don't think there is a requirement that CGs operate based on consensus

Reply:
Now says "support"